web-client/iron-svelte-client/src/lib/messages/message.svelte

```diff
@@ -8,10 +8,10 @@
             toastMessage = t.message;
             switch (t.type) {
                 case 'info':
-                    ui('#toast');
+                    // ui('#toast');
                     break;
                 case 'error':
-                    ui('#toast-error');
+                    // ui('#toast-error');
```

Comment:
Why is it commented?

Reply:
I should've mention this one earlier. 
the symbol ui is not imported, nor a global value. It was previously allowed by eslint and tsc because of non-strict rules. I tried to search what function named ui we have, but there is none, so I commented it out and thought you or Nicolas might know something about it.

Reply:
[beercss](https://github.com/beercss/beercss) must be updated to the v3.x version, and then you can import the `ui` function:
```js
import {ui} from "beercss";
```

Reply:
I fact, updating to v3.x version is breaking the UI, so we can’t fix this lint, let’s ignore it instead